cordova/build.sh

```diff
@@ -84,8 +84,14 @@ if [ ! -d $PROJECT ]; then
   cordova plugin add https://github.com/florentvaldelievre/virtualartifacts-webIntent.git
   checkOK
 
-  cordova plugin add https://github.com/phonegap/phonegap-plugin-barcodescanner.git
-  checkOK
+  if [ $CURRENT_OS != "WP8" ]
```

Comment:
I think that some platforms don't have $OSTYPE, so using $CURRENT_OS is more secure because it is passed from the command line